lua/pac3/editor/server/wear.lua

```diff
@@ -178,26 +190,8 @@ function pace.SubmitPart(data, filter)
 
 				pace.dupe_ents[key] = nil
 			end
-		elseif data.part then
-			pace.Parts[uid][data.part] = nil
-
-			-- this doesn't work because the unique id is different for some reason
```

Comment:
Big chunk of deadcode. This function is complicated enough, there's no reason to keep it. If someone wants to come back later and change it, this comment won't help them either.